src/screens/HomeScreen.tsx

```diff
@@ -39,16 +54,44 @@ export default function HomeScreen({navigation}: NativeStackHeaderProps) {
     }
   };
 
+  const muscleGroupData = () => {
+    console.log('muscleGroup', muscleGroup);
+    const temp = [];
+    for (const muscle of muscleGroup as any) {
+      if (muscle.name_en) {
+        temp.push({...muscle, key: muscle.id, value: muscle.name_en});
+      }
+    }
+    return temp;
+  };
+
+  console.log('muscleGroupData', muscleGroupData());
   return (
     <View>
-      <Header>Workout Generator</Header>
-      {/* replace navigation with modal? */}
-      <Button onPress={() => navigation.navigate('MuscleGroup')}>
-        <ButtonText>Generate</ButtonText>
+      <Header>{questions[1]}</Header>
+      <MultipleSelectList
+        setSelected={setSelected}
+        data={muscleGroupData()}
+        save="value"
+        boxStyles={{marginLeft: 20, marginRight: 20, borderRadius: 20}}
+        dropdownStyles={{marginLeft: 20, marginRight: 20, borderRadius: 20}}
```

Comment:
Can you create a style object for this so no inline styling

Reply:
Will complete this later in a separate style PR